<!doctype html>
<html>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, maximum-scale=1">
<title>Xie Lab</title>

<!-- website logo -->
<link rel="icon" href="img/LabLogo_v2.svg" type="image/svg">
<link href="css/bootstrap.min.css" rel="stylesheet" type="text/css">
<!-- 引入Fancybox样式表，用于创建图片和内容的弹出框效果 -->
<link rel="stylesheet" href="js/fancybox/jquery.fancybox.css" type="text/css" media="screen" />
<!-- 引入自定义样式表 -->
<link href="css/style.css" rel="stylesheet" type="text/css">
<link href="css/font-awesome.css" rel="stylesheet" type="text/css"> 
<!-- 引入Animate.css，用于动画效果 -->
<link href="css/animate.css" rel="stylesheet" type="text/css">
 
<!--[if lt IE 9]>
    <script src="js/respond-1.1.0.min.js"></script>
    <script src="js/html5shiv.js"></script>
    <script src="js/html5element.js"></script>
<![endif]-->

</head>
<body>
<div id="carousel" class="carousel slide carousel-fade" data-ride="carousel">
    <ol class="carousel-indicators">
        <li data-target="#carousel" data-slide-to="0" class="active"></li>
        <li data-target="#carousel" data-slide-to="1"></li>
        <li data-target="#carousel" data-slide-to="2"></li>
    </ol>
    <!--HOME settings -->
    <div class="carousel-inner">
        <div class="active item">
		<div class="caption zoomIn wow animated">
            <h2>Welcome to Xie Lab ! <br/><strong></strong></h2>
        </div>

        <img class="img-fluid" src="img/HOME2.jpg" alt=""></div>
        <div class="item">
		<div class="caption zoomIn wow animated">
              <!--<h2>HOBBY<br/><strong>Best Dreams Happen</strong></h2>-->
              </div>
			  <img class="img-fluid" src="img/Home-Fig1.jpg" alt=""></div>
        <div class="item">
		<div class="caption zoomIn wow animated">
              <!--<h2>MY WORLD<br/><strong>Take it before its too late</strong></h2>-->
             </div><img class="img-fluid" src="img/HOME3.jpg" alt=""></div>
    </div>
    <!-- Carousel nav -->
    <a class="carousel-control left" href="#carousel" data-slide="prev">&lsaquo;</a>
    <a class="carousel-control right" href="#carousel" data-slide="next">&rsaquo;</a>
</div>

<!--Header_section-->
<header id="header_wrapper">
  <div class="container">
    <div class="header_box">
      <!--<div class="logo"><a href="#">Resume</a></div>-->
	  <nav class="navbar navbar-inverse" role="navigation">
      <div class="navbar-header">
        <button type="button" id="nav-toggle" class="navbar-toggle" data-toggle="collapse" data-target="#main-nav"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
        </div>
	    <div id="main-nav" class="collapse navbar-collapse navStyle">
			<ul class="nav navbar-nav" id="mainNav">
			  <li class="active"><a href="#carousel" class="scroll-link">Home</a></li>
			  <li><a href="#research" class="scroll-link">RESEARCH</a></li>
              <li><a href="#members" class="scroll-link">MEMBERS</a></li>
			  <li><a href="#news" class="scroll-link">NEWS</a></li>
			  <li><a href="#resources" class="scroll-link">RESOURCES</a></li>
              <li><a href="#publications" class="scroll-link">PUBLICATIONS</a></li>
			  <li><a href="#contact" class="scroll-link">CONTACT</a></li>
			</ul>
      </div>
	 </nav>
    </div>
  </div>
</header>
<!--Research_section-->
<section id="research">
<div class="inner_wrapper about-us aboutUs-container fadeInLeft animated wow">
    <div class="container">
        <div class="section-title">
            <h2>RESEARCH</h2>
            <h6>By mass spectrometry to explore protein and RNA modifications and their biological roles.</h6>
        </div>
    </div>
    <div class="inner_section">
    <div class="research_content">
    <!-- 图片 -->
    <div class="image-container">
      <img src="img/Figure1.svg" alt="研究图片">
    </div>
    <!-- 文字内容 -->
    <div class="text-content">
        <!-- 标题 -->
      <div class="text-header">
        Comprehensive Characterization of Glycosylation Patterns Using Mass Spectrometry
      </div>
      <!-- 段落 -->
        <p>Glycans modify proteins, lipids, and even RNA molecules to form the glycocalyx, a dense regulatory coat on cell surfaces that plays a crucial role in cellular functions. Changes in glycosylation patterns correlate closely with the development and further disease progression. Consequently, the profiling of glycans in biology and medicine is an important approach. This advance regarding glycan analysis, therefore, has been underpinned by mass spectrometry techniques. These allow high-throughput, sensitive, and precise characterization of both composition and structure. </p>
        <p>In our lab, we focus on developing state-of-the-art mass spectrometry to enable quantitative structural identification of glycosylation on proteins and RNA molecules. This includes the development of comprehensive workflows, which are tightly integrating advanced sample preparation with data acquisition and computational analysis into unprecedented accuracy and depth in glycosylation profiling. It is in this backdrop that our efforts have gone toward providing an overarching system explaining how different glycan architectures influence molecular interactions, intracellular signaling, and pathogenic mechanisms and thereby link glycomics to functional biology.</p>
    </div>
    </div>
    <div class="research_content">
    <!-- 文字内容 -->
    <div class="text-content">
    <!-- 标题 -->
    <div class="text-header">
    Unraveling Glycan-Mediated Interaction Networks via Chemical Biology Approaches
    </div>
    <!-- 段落 -->
        <p>Biological interaction networks play a central role in deciphering complicated molecular machinery and signal transduction pathways active in the cell, affecting development and disease course. Biomolecules are frequently modified, and such modifications determine their functions in various cellular contexts. Of those, glycosylation, which mediates the formation of the glycocalyx, presents several of the biggest challenges to inquiry. So far, the role that various glycan structures play in modulating membrane interaction networks has been only poorly explored, despite their widely recognized key role in cellular signaling and recognition.</p>
        <p>In concert with this limitation, we have established expertise in a set of chemical biology toolbox, including proximity labeling, crosslinking mass spectrometry, and affinity purification mass spectrometry, which enable construct of the networks of glycan-mediated interactions at far superior levels of specificity and sensitivity. Using these sophisticated approaches, we aim to achieve a comprehensive molecular understanding of how glycosylation regulates major biological processes. The ultimate goal is to explain how glycosylation acts at the cellular level to change cellular functions and potentially uncover new therapeutic interventions in diseases caused by glycosylation dysregulation.</p>
    </div>
          <!-- 图片 -->
    <div class="image-container" style="margin-right:0 !important;margin-left: 30px !important;">
    <img src="img/Figure2.svg" alt="研究图片">
    </div>

    </div>
    <div class="research_content">
        <!-- 图片 -->
        <div class="image-container">
        <img src="img/Figure3.svg" alt="研究图片">
        </div>
        <!-- 文字内容 -->
        <div class="text-content">
        <!-- 标题 -->
        <div class="text-header">
        Quantitative Mass Spectrometry-Based Epitranscriptomics
        </div>
        <!-- 段落 -->
            <p>RNA participates directly in events related to both gene expression and regulation in organisms. More than 150 different chemical modifications have been identified in RNA that participate in significant functions, including mRNA stability, RNA conformation, translation efficiency, and splicing. RNA modifications have been implicated in a variety of diseases, including cancers and neurological disorders, through their dysregulation, underlining their critical relevance to cellular physiology and disease mechanisms. Conventional genomic methods have various limitations in simultaneously characterizing and quantifying multiple RNA modifications, which makes mass spectrometry a unique tool for these tasks. Mass spectrometry enables direct identification of modified entities through their specific mass shifts and provides accurate quantitative information.</p>
            <p>Our laboratory is highly motivated to advance mass spectrometry-based methodologies for the high-throughput and quantitative characterization of RNA modifications. Through the creation of novel mass spectrometry techniques, we intend to investigate the dynamics of RNA modifications across different disease conditions, thereby addressing significant gaps in the understanding of RNA biology. This research aspires to uncover the regulatory roles of RNA modifications in gene expression, providing valuable insights into epitranscriptomic processes and identifying prospective biomarkers and therapeutic targets.</p>
        </div>
</div>

    </div>
</div>
</section>
<!--Members-->
<section id="members">
    <div class="inner_wrapper about-us aboutUs-container fadeInLeft animated wow">
        <div class="container">
            <div class="section-title">
                <h2 style="color: white">Lab Members</h2>
                <h6>Thanks to all members of the Xie lab for their devotion and contribution!</h6>
            </div>
            <div class="pi-section">
                <div class="pi-profile">
                    <img src="img/Xie.JPG" alt="PI Photo" class="pi-photo">
                    <div class="pi-info">
                        <h3 class="pi-name">Yixuan (Axe) Xie 谢一轩</h3>
                        <ul class="experience-list">
                            <li>2024–present: Greater Bay Area Institute of Precision Medicine, Fudan University (Principal Investigator)</li>
                            <li>2023–2024: Washington University School of Medicine (Staff Scientist)</li>
                            <li>2020–2023: Washington University School of Medicine & Perelman School of Medicine at the University of Pennsylvania (Postdoctoral Researcher)</li>
                            <li>2016–2020: University of California, Davis (Ph.D.)</li>
                            <li>2014–2016: The University of Akron (M.S.)</li>
                            <li>2011–2015: Lanzhou University (B.S.)</li>
                        </ul>
                        <div class="pi-email">
                        <p>Email: <a href="mailto:xieyixuan@ipm.gba.org.cn">xieyixuan@ipm.gba.org.cn</a></p>
                        <p>Academic Profiles:
                            <a href="https://www.ipm-gba.org.cn/Index/team_details/id/549.html" target="_blank" rel="noopener noreferrer">Greater Bay Area Institute of Precision Medicine</a> |
                            <a href="https://scholar.google.com/citations?hl=en&user=zTN_b0kAAAAJ" target="_blank" rel="noopener noreferrer">Google Scholar</a>
                        </p>
                        </div>
                    </div>
                </div>
            </div>
            <div class="pi-section">
                <h3 style="color: #FEE16F"> Current Members</h3>
                <div class="student-profile">
                    <div>
                        <img src="img/eva.JPG" alt="Student 1" class="pi-photo">
                        <div class="student-name">
                            <h4>Li Yi 伊力 (Ph.D. Student)</h4>
                            <div>
                                <p class="research-tag1">Epitranscriptomics</p>
                                <p class="research-tag2">Bioinformatics</p>
                            </div>
                            <p>Email: <a href="mailto:lyi24@m.fudan.edu.cn">lyi24@m.fudan.edu.cn</a></p>
                        </div>
                    </div>

                </div>
            </div>
            <h3 style="color: #FEE16F; text-align: center;margin-bottom: 30px;"> Animal Crossing </h3>
            <div class="pi-section">
                <!--Paw Patrol;Fluffy Crew;Critter Gang;Housemates-->
                <div class="student-profile">
                    <div>
                        <img src="img/YiYi.JPG" alt="Student 1" class="pi-photo">
                        <div class="student-name">
                            <h4>Yi Yi (Ph.D.)</h4>
                            <div>
                                <p class="research-tag1">intelligent</p>
                                <p class="research-tag2">energetic</p>
                                <p class="research-tag3">ball-obsessed</p>
                            </div>
                        </div>
                    </div>
                    <div>
                        <img src="img/bao.jpg" alt="Student 2" class="pi-photo">
                        <div class="student-name">
                            <h4>Bao</h4>
                            <div>
                                <p class="research-tag2">Adorable little mischief maker</p>
                            </div>
                        </div>
                    </div>
                    <div>
                        <img src="img/banli.jpg" alt="Student 3" class="pi-photo">
                        <div class="student-name">
                            <h4>Lee</h4>
                            <div>
                                <p class="research-tag1">Shadow</p>
                                <p class="research-tag2">Hunter</p>
                                <p class="research-tag3">Nap Master</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div>

        </div>
    </div>
</section>
<!-- Portfolio -->
<section id="news" class="content">

  <!-- Container -->
  <div class="container portfolio_title">

    <!-- Title -->
    <div class="section-title">
        <h2>News</h2>
        <h6>Enjoy the moment</h6>
    </div>
    <!--/Title -->

  </div>
  <!-- Container -->

  <div class="portfolio-top"></div>

  <!-- Portfolio Filters -->
  <div class="portfolio">

    <div id="filters" class="sixteen columns">
      <ul class="clearfix">
        <li><a id="all" href="#" data-filter="*" class="active">
          <h5>All</h5>
          </a></li>
        <li><a class="" href="#" data-filter=".Meetings">
          <h5>Meetings</h5>
          </a></li>
        <li><a class="" href="#" data-filter=".People">
          <h5>People</h5>
          </a></li>
        <li><a class="" href="#" data-filter=".Publications">
          <h5>Publications</h5>
          </a></li>
        <li><a class="" href="#" data-filter=".Team">
          <h5>Team Retreat</h5>
          </a></li>
      </ul>
    </div>
    <!--/Portfolio Filters -->

    <!-- Portfolio Wrapper -->
    <div class="isotope fadeInLeft animated wow grid" style="position: relative; overflow: hidden; height: 480px;" id="portfolio_wrapper">
      <!-- Portfolio Item -->
      <figure style="position: absolute; left: 0px; top: 0px; transform: translate3d(0px, 0px, 0px) scale3d(1, 1, 1); width: 337px; opacity: 1;" class="portfolio-item one-four   Meetings isotope-item effect-oscar">

        <div class="portfolio_img">
		<img src="img/pics/202411zhanjiang.JPG"  alt="Portfolio 1"> </div>
			<figcaption>
				<div>
				  <a href="img/pics/202411zhanjiang.JPG" class="fancybox">
					<h2>2024.11.28 <span>湛江</span></h2>
							<p>《全国糖科学与糖工程学术会议暨产业论坛》</p>
				  </a>
				</div>
			</figcaption>
        </figure>
      <!--/Portfolio Item -->

      <!-- Portfolio Item-->
      <figure style="position: absolute; left: 0px; top: 0px; transform: translate3d(337px, 0px, 0px) scale3d(1, 1, 1); width: 337px; opacity: 1;" class="portfolio-item one-four  Meetings isotope-item effect-oscar">
        <div class="portfolio_img"> <img src="img/pics/202409nanjing.JPG" alt="Portfolio 1"> </div>
        	<figcaption>
				<div>
				  <a href="img/pics/202409nanjing.JPG" class="fancybox">
					<h2>2024.09.21 <span>南京</span></h2>
							<p>《表观遗传前沿理论与方法学术论坛》</p>
				  </a>
				</div>
			</figcaption>
        </figure>
      <!--/Portfolio Item -->

      <!-- Portfolio Item -->
      <figure style="position: absolute; left: 0px; top: 0px; transform: translate3d(674px, 0px, 0px) scale3d(1, 1, 1); width: 337px; opacity: 1;" class="portfolio-item one-four  People  isotope-item effect-oscar">
        <div class="portfolio_img"> <img src="img/pics/eva.jpg" alt="Portfolio 1"> </div>
       <figcaption>
				<div>
				  <a href="img/pics/eva.jpg" class="fancybox">
					<h2>2024.09.11 <span></span></h2>
							<p>Welcome new members,Li Yi !</p>
				  </a>
				</div>
			</figcaption>
        </figure>
      <!--/Portfolio Item-->

      <!-- Portfolio Item-->
      <figure style="position: absolute; left: 0px; top: 0px; transform: translate3d(1011px, 0px, 0px) scale3d(1, 1, 1); width: 337px; opacity: 1;" class="portfolio-item one-four  People  nature web isotope-item effect-oscar">
        <div class="portfolio_img"> <img src="img/pics/xiexie.JPG" alt="Portfolio 1"> </div>
         <figcaption>
				<div>
				  <a href="img/pics/xiexie.JPG" class="fancybox">
					<h2>2024.09.09 <span></span></h2>
							<p>The grand founding day of the research group</p>
				  </a>
				</div>
			</figcaption>
      </figure>
      <!-- Portfolio Item -->

      <!-- Portfolio Item-->
      <figure style="position: absolute; left: 0px; top: 0px; transform: translate3d(1011px, 0px, 0px) scale3d(1, 1, 1); width: 337px; opacity: 1;" class="portfolio-item one-four  Team  nature web isotope-item effect-oscar">
        <div class="portfolio_img"> <img src="img/pics/group1.JPG" alt="Portfolio 1"> </div>
         <figcaption>
				<div>
				  <a href="img/pics/group1.JPG" class="fancybox">
					<h2>2024.08.26 <span></span></h2>
							<p>Meeting Day</p>
				  </a>
				</div>
			</figcaption>
      </figure>
      <!-- Portfolio Item -->

    </div>
    <!--/Portfolio Wrapper -->

  </div>
  <!--/Portfolio Filters -->

  <div class="portfolio_btm"></div>


  <div id="project_container">
    <div class="clear"></div>
    <div id="project_data"></div>
  </div>


</section>
<!--/Portfolio -->

<!--Resources-->
<section  id="resources">
  <div class="container">
    <h2 style="color: white">Software/Database</h2>
	<h6>All resources are available to download for free</h6>
    <div class="service_wrapper">
      <div class="row_left">
        <div class="col-md-3">
		<div class="service_icon delay-03s animated wow  zoomIn"> <span><i class="fa fa-tree"></i></span> </div>
        <div class="service_block">
          <a target="_blank" rel="noopener noreferrer" href="https://glycointeractome.org/" style="text-decoration: none;">
            <h3 class="animated fadeInUp wow" style="color: orange; white-space: nowrap;">Glycan-interactome</h3>
            <p class="animated fadeInDown wow">The glycan-dependent protein interactome (GDPI) across different glycan phenotypes.</p>
          </a>
        </div>
        </div>
        <div class="col-md-3"> 
		<div class="service_icon icon2  delay-03s animated wow zoomIn"> <span><i class="fa fa-tree"></i></span> </div>
		<div class="service_block">
            <a target="_blank" rel="noopener noreferrer" href=" https://github.com/ChenfengZhao/NuMoFinder" style="text-decoration: none;">
            <h3 class="animated fadeInUp wow"  style="color: orange; white-space: nowrap;">NuMoFinder</h3>
            <p class="animated fadeInDown wow">A tool to identify and quantify the peaks of nucleic acid modifications referred from Modomics and DNAmod databases</p>
            </a>
        </div>
        </div>
          <div class="col-md-3">
		<div class="service_icon delay-03s animated wow  zoomIn"> <span><i class="fa fa-tree"></i></span> </div>
        <div class="service_block">
          <a target="_blank" rel="noopener noreferrer" href="https://github.com/ChenfengZhao/GlycanDIAFinder" style="text-decoration: none;">
            <h3 class="animated fadeInUp wow" style="color: orange; white-space: nowrap;">GlycanDIAFinder</h3>
            <p class="animated fadeInDown wow">Its aim is to analyze Glycosylation with the sensitive data-independent acquisition (DIA) strategy.</p>
          </a>
        </div>
        </div>
          <div class="col-md-3">
		<div class="service_icon delay-03s animated wow  zoomIn"> <span><i class="fa fa-tree"></i></span> </div>
        <div class="service_block">
          <a target="_blank" rel="noopener noreferrer" href="https://github.com/zfyuan/EpiProfile2.0_Family" style="text-decoration: none;">
            <h3 class="animated fadeInUp wow" style="color: orange; white-space: nowrap;">EpiProfile</h3>
            <p class="animated fadeInDown wow">A Computational Platform for Processing Epi-Proteomics Mass Spectrometry Data</p>
          </a>
        </div>
        </div>
        <!--<div class="col-md-3">
		<div class="service_icon icon3  delay-03s animated wow zoomIn"> <span><i class="fa fa-building-o "></i></span> </div>
          <div class="service_block">
            
            <h3 class="animated fadeInUp wow">Lighting</h3>
            <p class="animated fadeInDown wow">Lorem Ipsum is simply dummy text of the printing and typesetting industry.  </p>
          </div>
        </div>
		  <div class="col-md-3">
		<div class="service_icon icon3  delay-03s animated wow zoomIn"> <span><i class="fa fa-twitter"></i></span> </div>
          <div class="service_block">
            
            <h3 class="animated fadeInUp wow">Dimensions</h3>
            <p class="animated fadeInDown wow">Lorem Ipsum is simply dummy text of the printing and typesetting industry.  </p>
          </div>
        </div>-->
      </div> 
	 
    </div>
  </div>
</section>
 <!--/Publications -->
<section id="publications">
    <div class="container">
        <div class="section-title">
            <h2 style="margin-bottom: 20px !important;">Latest Publications</h2>
            <h6>Xie Lab Publications from 2017 to 2024</h6>
        </div>
        <div class="row" style="display: flex;">
            <div class="timeline" id="timeline">
                <div onclick="showYear('all')" class="year-button">ALL</div>
                <div onclick="showYear('2024')" class="year-button">2024</div>
                <!--<div onclick="showYear('2023')" class="year-button">2023</div>-->
                <div onclick="showYear('2017')" class="year-button">Prior to Independent Research</div>
            </div>
            <div class="pub_content">
            <div class="year" id="2024">
                <div class="custom-heading-year">2024</div>
                <div class="publication">
                    <p>
                        Xie, Y., Liu, X., Zhao, C., Chen, S., Wang, S., Lin, Z., Robison, F. M., George, B. M., Flynn, R. A., Lebrilla, C. B., & Garcia, B. A. (2024),
                        <a href="https://doi.org/10.1101/2024.03.12.584702" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                            <span style="color: rgb(23, 54, 93);"><em><strong> Development and application of GlycanDIA workflow for glycomic analysis.</strong></em></span>
                        </a>
                        bioRxiv: the preprint server for biology, 2024.03.12.584702.
                    </p>
                </div>
                <div class="publication">
                    <p>
                        Liu, X., Yi, L., Lin, Z., Chen, S., Wang, S., Sheng, Y., Lebrilla, C. B., Garcia, B. A., & Xie, Y. (2024),
                        <a href="https://doi.org/10.1101/2024.10.30.621210" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                            <span style="color: rgb(23, 54, 93);"><em><strong> Metabolic control of glycosylation forms for establishing glycan-dependent protein interaction networks.</strong></em></span>
                        </a>
                        bioRxiv (Cold Spring Harbor Laboratory).
                    </p>
                </div>
                <div class="publication">
                    <p>
                        Xie, Y., Brás-Costa, C., Lin, Z., & Garcia, B. A. (2024),
                        <a href="https://doi.org/10.1002/mas.21907" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                            <span style="color: rgb(23, 54, 93);"><em><strong> Mass spectrometry analysis of nucleic acid modifications: from beginning to future.</strong></em></span>
                        </a>
                        Mass spectrometry reviews, Advance online publication.
                    </p>
                </div>
            </div>
            <div class="year" id="2017">
                <div class="custom-heading-year">Prior to Independent Research</div>
                <div class="publication">
                    <p>Xie, Y., Chai, P., Till, N. A., Hemberger, H., Lebedenko, C. G., Porat, J., Watkins, C. P., Caldwell, R. M., George, B. M., Perr, J., Bertozzi, C. R., Garcia, B. A., & Flynn, R. A. (2024),<a href="https://doi.org/10.1016/j.cell.2024.07.044" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong> The modified RNA base acp3U is an attachment site for N-glycans in glycoRNA.</strong></em></span></a> Cell, 187(19), 5228–5237.e12.</p>
                </div>
                <div class="publication">
                    <p>Porat, J., Watkins, C. P., Jin, C., Xie, Y., Tan, X., Lebedenko, C. G., Hemberger, H., Shin, W., Chai, P., Collins, J. J., Garcia, B. A., Bojar, D., & Flynn, R. A. (2024),<a href="https://doi.org/10.1101/2024.08.28.610074" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong> O-glycosylation contributes to mammalian glycoRNA biogenesis.</strong></em></span></a> bioRxiv: the preprint server for biology, 2024.08.28.610074.</p>
                </div>
                <div class="publication">
                    <p>Liu, X., Zhang, Y., Wen, Z., Hao, Y., Banks, C. A. S., Cesare, J., Bhattacharya, S., Arvindekar, S., Lange, J. J., Xie, Y., Garcia, B. A., Slaughter, B. D., Unruh, J. R., Viswanath, S., Florens, L., Workman, J. L., & Washburn, M. P. (2024),<a href="https://doi.org/10.1073/pnas.2318601121" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong> An integrated structural model of the DNA damage-responsive H3K4me3 binding WDR76:SPIN1 complex with the nucleosome.</strong></em></span></a> Proceedings of the National Academy of Sciences of the United States of America, 121(33), e2318601121.</p>
                </div>
                <div class="publication">
                    <p> Lin, Z., Xie, Y., Gongora, J., Liu, X., Zahn, E., Palai, B. B., Ramirez, D., Searfoss, R. M., Vitorino, F. N., Dann, G. P., Zhao, C., Han, X., MacTaggart, B., Lan, X., Fu, D., Greenberg, L., Zhang, Y., Lavine, K. J., Greenberg, M. J., Lv, D., … Garcia, B. A. (2024),<a href="https://doi.org/10.1101/2024.06.01.596974" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong> An unbiased proteomic platform for activity-based arginylation profiling.</strong></em></span></a> bioRxiv : the preprint server for biology, 2024.06.01.596974.</p>
                </div>
                <div class="publication">
                    <p> Izadifar, Z., Cotton, J., Chen, S., Horvath, V., Stejskalova, A., Gulati, A., LoGrande, N. T., Budnik, B., Shahriar, S., Doherty, E. R., Xie, Y., To, T., Gilpin, S. E., Sesay, A. M., Goyal, G., Lebrilla, C. B., & Ingber, D. E. (2024),<a href="https://doi.org/10.1038/s41467-024-48910-0" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong> Mucus production, host-microbiome interactions, hormone sensitivity, and innate immune responses modeled in human cervix chips.</strong></em></span></a> Nature communications, 15(1), 4578.</p>
                </div>
                <div class="publication">
                    <p>Xie, Y., Chen, S., Alvarez, M. R., Sheng, Y., Li, Q., Maverakis, E., & Lebrilla, C. B. (2024),
                        <a href="https://doi.org/10.1039/d3sc06432h" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                            <span style="color: rgb(23, 54, 93);"><em><strong> Protein oxidation of fucose environments (POFE) reveals fucose-protein interactions.</strong></em></span>
                        </a>
                        Chemical Science, 15(14), 5256–5267.
                    </p>
                </div>
                <div class="publication">
                    <p>
                        Frisbie, V. S., Hashimoto, H., Xie, Y., De Luna Vitorino, F. N., Baeza, J., Nguyen, T., Yuan, Z., Kiselar, J., Garcia, B. A., & Debler, E. W. (2024),
                        <a href="https://doi.org/10.1038/s41467-024-46637-6" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                            <span style="color: rgb(23, 54, 93);"><em><strong> Two DOT1 enzymes cooperatively mediate efficient ubiquitin-independent histone H3 lysine 76 tri-methylation in kinetoplastids.</strong></em></span>
                        </a>
                        Nature communications, 15(1), 2467.
                    </p>
                </div>
                <div class="publication">
                <p>Lin, Z., Gongora, J., Liu, X., Xie, Y., Zhao, C., Lv, D., & Garcia, B. A. (2023),
                <a href="https://doi.org/10.1021/acs.jproteome.3c00467" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                <span style="color: rgb(23, 54, 93);"><em><strong>Automation to enable high-throughput chemical proteomics.</strong></em></span></a>
                Journal of Proteome Research, 22(12), 3676–3682.</p>
            </div>
                <div class="publication">
                <p>Lu-Culligan, W. J., Connor, L. J., Xie, Y., Ekundayo, B. E., Rose, B. T., Machyna, M., Pintado-Urbanc, A. P., Zimmer, J. T., Vock, I. W., Bhanu, N. V., King, M. C., Garcia, B. A., Bleichert, F., & Simon, M. D. (2023),
                <a href="https://doi.org/10.1038/s41586-023-06565-9" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                <span style="color: rgb(23, 54, 93);"><em><strong>Acetyl-methyllysine marks chromatin at active transcription start sites.</strong></em></span></a>
                Nature, 622(7981), 173–179.</p>
            </div>
                <div class="publication">
                <p>Chen, S., Bouchibti, Y., Xie, Y., Chen, Y., Chang, V., & Lebrilla, C. B. (2023),
                <a href="https://doi.org/10.1021/acs.analchem.3c02230" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;">
                <span style="color: rgb(23, 54, 93);"><em><strong>Analysis of cell glycogen with quantitation and determination of branching using liquid chromatography-mass spectrometry.</strong></em></span></a>
                Analytical Chemistry, 95(34), 12884–12892.</p>
            </div>
                <div class="publication">
                    <p>Lauman, R., Kim, H. J., Pino, L. K., Scacchetti, A., Xie, Y., Robison, F., Sidoli, S., Bonasio, R., & Garcia, B. A. (2023). <a href="https://doi.org/10.1021/acs.analchem.2c04114" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Expanding the epitranscriptomic RNA sequencing and modification mapping mass spectrometry toolbox with field asymmetric waveform ion mobility and electrochemical elution liquid chromatography.</strong></em></span></a> Analytical Chemistry, 95(12), 5187–5195.</p>
                </div>
                <div class="publication">
                    <p>Alvarez, M. R., Zhou, Q., Tena, J., Barboza, M., Wong, M., Xie, Y., Lebrilla, C. B., Cabanatan, M., Barzaga, M. T., Tan-Liu, N., Heralde, F. M., 3rd, Serrano, L., Nacario, R. C., & Completo, G. C. (2023). <a href="https://doi.org/10.3390/cancers15051559" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Glycomic, glycoproteomic, and proteomic profiling of philippine lung cancer and peritumoral tissues: case series study of patients stages I–III.</strong></em></span></a> Cancers, 15(5), 1559.</p>
                </div>
                <div class="publication">
                    <p>Xie, Y., De Luna Vitorino, F. N., Chen, Y., Lempiäinen, J. K., Zhao, C., Steinbock, R. T., Lin, Z., Liu, X., Zahn, E., Garcia, A. L., Weitzman, M. D., & Garcia, B. A. (2023). <a href="https://doi.org/10.1039/d3cc04402e" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>SWAMNA: a comprehensive platform for analysis of nucleic acid modifications.</strong></em></span></a> Chemical Communications (Cambridge, England), 59(83), 12499–12502.</p>
                </div>
                <div class="publication">
                    <p>Merleev, A. A., Le, S. T., Alexanian, C., Toussi, A., Xie, Y., Marusina, A. I., Watkins, S. M., Patel, F., Billi, A. C., Wiedemann, J., Izumiya, Y., Kumar, A., Uppala, R., Kahlenberg, J. M., Liu, F. T., Adamopoulos, I. E., Wang, E. A., Ma, C., Cheng, M. Y., Xiong, H., … Maverakis, E. (2022), <a href="https://doi.org/10.1172/jci.insight.159762" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Biogeographic and disease-specific alterations in epidermal lipid composition and single-cell analysis of acral keratinocytes.</strong></em></span></a> JCI Insight, 7(16), e159762.</p>
                </div>
                <div class="publication">
                    <p>Xie, Y., Janssen, K. A., Scacchetti, A., Porter, E. G., Lin, Z., Bonasio, R., & Garcia, B. A. (2022), <a href="https://doi.org/10.1021/acs.analchem.2c00471" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Permethylation of ribonucleosides provides enhanced mass spectrometry quantification of post-transcriptional RNA modifications.</strong></em></span></a> Analytical Chemistry, 94(20), 7246–7254.</p>
                </div>
                <div class="publication">
                    <p>Janssen, K. A., Xie, Y., Kramer, M. C., Gregory, B. D., & Garcia, B. A. (2022), <a href="https://doi.org/10.1021/jasms.2c00065" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Data-independent acquisition for the detection of mononucleoside RNA modifications by mass spectrometry.</strong></em></span></a> Journal of the American Society for Mass Spectrometry, 33(5), 885–893.</p>
                </div>
                <div class="publication">
                    <p>Sheng, Y., Vinjamuri, A., Alvarez, M. R. S., Xie, Y., McGrath, M., Chen, S., Barboza, M., Frieman, M., & Lebrilla, C. B. (2022), <a href="https://doi.org/10.3389/fmolb.2022.799703" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Host cell glycocalyx remodeling reveals SARS-CoV-2 Spike protein glycomic binding sites.</strong></em></span></a> Frontiers in Molecular Biosciences, 9, 799703.</p>
                </div>
                <div class="publication">
                    <p>Li, Q., Xie, Y., Rice, R., Maverakis, E., & Lebrilla, C. B. (2022), <a href="https://doi.org/10.1039/d1sc06898a" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>A proximity labeling method for protein-protein interactions on cell membrane.</strong></em></span></a> Chemical Science, 13(20), 6028–6038.</p>
                </div>
                <div class="publication">
                    <p>Kim, T., Xie, Y., Li, Q., Artegoitia, V. M., Lebrilla, C. B., Keim, N. L., Adams, S. H., & Krishnan, S. (2021), <a href="https://doi.org/10.1007/s00394-021-02539-7" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Diet affects glycosylation of serum proteins in women at risk for cardiometabolic disease.</strong></em></span></a> European Journal of Nutrition, 60(7), 3727–3741.</p>
                </div>
                <div class="publication">
                    <p>Zhou, Q., Xie, Y., Lam, M., & Lebrilla, C. B. (2021), <a href="https://doi.org/10.3390/cells10092318" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>N-Glycomic analysis of the cell shows specific effects of glycosyl transferase inhibitors.</strong></em></span></a> Cells, 10(9), 2318.</p>
                </div>
                <div class="publication">
                    <p>Maverakis, E., Merleev, A. A., Park, D., Kailemia, M. J., Xu, G., Ruhaak, L. R., Kim, K., Hong, Q., Li, Q., Leung, P., Liakos, W., Wan, Y. Y., Bowlus, C. L., Marusina, A. I., Lal, N. N., Xie, Y., Luxardi, G., & Lebrilla, C. B. (2021), <a href="https://doi.org/10.1016/j.clim.2021.108825" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Glycan biomarkers of autoimmunity and bile acid-associated alterations of the human glycome: primary biliary cirrhosis and primary sclerosing cholangitis-specific glycans.</strong></em></span></a> Clinical Immunology, 230, 108825.</p>
                </div>
                <div class="publication">
                    <p>Zheng, J. J., Agus, J. K., Hong, B. V., Tang, X., Rhodes, C. H., Houts, H. E., Zhu, C., Kang, J. W., Wong, M., Xie, Y., Lebrilla, C. B., Mallick, E., Witwer, K. W., & Zivkovic, A. M. (2021), <a href="https://doi.org/10.1038/s41598-021-95451-3" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Isolation of HDL by sequential flotation ultracentrifugation followed by size exclusion chromatography reveals size-based enrichment of HDL-associated proteins.</strong></em></span></a> Scientific Reports, 11(1), 16086.</p>
                </div>
                <div class="publication">
                    <p>Xie, Y., Chen, S., Li, Q., Sheng, Y., Alvarez, M. R., Reyes, J., Xu, G., Solakyildirim, K., & Lebrilla, C. B. (2021), <a href="https://doi.org/10.1039/d1sc00814e" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Glycan-protein cross-linking mass spectrometry reveals sialic acid-mediated protein networks on cell surfaces.</strong></em></span></a> Chemical Science, 12(25), 8767–8777.</p>
                </div>
                <div class="publication">
                    <p>Kasper, D. M., Hintzen, J., Wu, Y., Ghersi, J. J., Mandl, H. K., Salinas, K. E., Armero, W., He, Z., Sheng, Y., Xie, Y., Heindel, D. W., Park, E. J., Sessa, W. C., Mahal, L. K., Lebrilla, C., Hirschi, K. K., & Nicoli, S. (2020), <a href="https://doi.org/10.1126/science.aaz2121" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>The N-glycome regulates the endothelial-to-hematopoietic transition.</strong></em></span></a> Science, 370(6521), 1186–1191.</p>
                </div>
                <div class="publication">
                    <p>Merleev, A. A., Park, D., Xie, Y., Kailemia, M. J., Xu, G., Ruhaak, L. R., Kim, K., Hong, Q., Li, Q., Patel, F., Wan, Y. Y., Marusina, A. I., Adamopoulos, I. E., Lal, N. N., Mitra, A., Le, S. T., Shimoda, M., Luxardi, G., Lebrilla, C. B., & Maverakis, E. (2020), <a href="https://doi.org/10.1038/s41598-020-73588-x" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>A site-specific map of the human plasma glycome and its age and gender-associated alterations.</strong></em></span></a> Scientific Reports, 10(1), 17505.</p>
                </div>
                <div class="publication">
                    <p>Park, S., Doherty, E. E., Xie, Y., Padyana, A. K., Fang, F., Zhang, Y., Karki, A., Lebrilla, C. B., Siegel, J. B., & Beal, P. A. (2020), <a href="https://doi.org/10.1038/s41467-020-18862-2" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>High-throughput mutagenesis reveals unique structural features of human ADAR1.</strong></em></span></a> Nature Communications, 11(1), 5130.</p>
                </div>
                <div class="publication">
                    <p>Li, Q., Xie, Y., Wong, M., Barboza, M., & Lebrilla, C. B. (2020), <a href="https://doi.org/10.1038/s41596-020-0350-4" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Comprehensive structural glycomic characterization of the glycocalyxes of cells and tissues.</strong></em></span></a> Nature Protocols, 15(8), 2668–2704.</p>
                </div>
                <div class="publication">
                    <p>Xie, Y., Sheng, Y., Li, Q., Ju, S., Reyes, J., & Lebrilla, C. B. (2020), <a href="https://doi.org/10.1039/d0sc04199h" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><span style="color: rgb(23, 54, 93);"><em><strong>Determination of the glycoprotein specificity of lectins on cell membranes through oxidative proteomics.</strong></em></span></a> Chemical Science, 11(35), 9501–9512.</p>
                </div>
                <div class="publication">
                    <p>Li, Q., Xie, Y., Wong, M., & Lebrilla, C. B. (2019). <a href="https://doi.org/10.3390/cells8080882" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><em><strong>Characterization of cell glycocalyx with mass spectrometry methods.</strong></em></a> Cells, 8(8), 882.</p>
                </div>
                <div class="publication">
                    <p>Li, Q., Xie, Y., Xu, G., & Lebrilla, C. B. (2019). <a href="https://doi.org/10.1039/c9sc01360a" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><em><strong>Identification of potential sialic acid binding proteins on cell membranes by proximity chemical labeling.</strong></em></a> Chemical Science, 10(24), 6199–6209.</p>
                </div>
                <div class="publication">
                    <p>Xie, Y., Peng, C., Gao, Y., Liu, X., Liu, T., & Joy, A. (2017). <a href="https://doi.org/10.1002/pola.28776" target="_self" style="color: rgb(23, 54, 93); text-decoration: underline;"><em><strong>Mannose-based graft polyesters with tunable binding affinity to concanavalin A.</strong></em></a> Journal of Polymer Science Part A: Polymer Chemistry, 55, 3908-3917.</p>
                </div>

        </div>
            </div>
        </div>
    </div>
</section>

<!--Contact-->
<footer class="footer_wrapper" id="contact">
    <div class="container">
        <section class="page_section contact" >
            <div class="contact_section">
                <h2>Available Positions</h2>
                <h6>If you are interested in our research, feel free to contact us anytime</to></h6>
            </div>
            <div class="row" style="display: flex;">
                <div style="display: flex; padding: 20px; border-radius: 8px; box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1); background-color: #333;">
                    <!-- Left Content -->
                    <div class="left-content" style="flex: 1; padding-right: 20px; border-right: 1px solid #ddd; padding: 20px;">
                        <div class="contact_text" style="margin-bottom: 30px;">
                            <h4>Postdoctoral Researchers</h4>
                            <p>We are seeking outstanding postdoctoral researchers interested in a stimulating training environment to expand their expertise in mass spectrometry and glycosylation research. We welcome postdoctoral candidates with diverse training backgrounds, including but not limited to: Mass Spectrometry, Glycobiology, Epitranscriptomics, Epigenetics, Biochemistry, and Molecular Biology. Interested candidates should submit their CV, including the names and contact information for two references, along with a brief description of their doctoral training and research goals, to Dr. Yixuan Xie (xieyixuan@ipm-gba.org.cn).</p>
                            <a target="_blank" style="color: #ADD8E6;font-size: 1.2em; text-decoration: none; font-weight: bold;" href="https://mp.weixin.qq.com/s/MKQDoRcAYFAynIt6gpOV8Q">
                                <span>More information</span>
                            </a>
                        </div>
                    </div>
                    <!-- Right Content -->
                    <div class="right-align" style="flex: 1; padding-left: 20px;padding: 20px;">
                        <div class="contact_text" style="margin-bottom: 30px;">
                            <h4>Graduate/Undergraduate Students</h4>
                            <p>We are also looking for motivated graduate students with a passion for applying mass spectrometry-based technologies to solve complex biological questions related to glycosylation. At the same time, limited research positions are available for undergraduate students interested in summer or academic-year research projects. Interested students can email Dr. Yixuan Xie (xieyixuan@ipm-gba.org.cn) to inquire about potential projects and the opportunity to join the lab.</p>
                        </div>
                    </div>
                </div>
            </div>
            <div class="address">
                <h4 style="color: #fff; font-size: 1.5em; margin-bottom: 10px;"><span class="icon">🏢</span>&nbsp;Address</h4>
                <p style="color: #aaa; font-size: 1em; line-height: 1.5;">
                    <a href="https://www.ipm-gba.org.cn/" target="_blank" rel="noopener noreferrer" style="color: #ADD8E6; text-decoration: none;">
            State Key Laboratory of Genetic Engineering,<br>
            Greater Bay Area Institute of Precision Medicine (Guangzhou),<br>
            School of Life Sciences and Institutes of Biomedical Sciences,<br>
            Fudan University,<br>
            Shanghai,<br>
            China
                                    </a>
                </p>
            </div>
        </section>
  </div>
  <div class="container">
    <div class="footer_bottom"><span>Copyright 2024 Glyco Xie Lab. All rights reserved. </span> </div>
  </div>
</footer>

<script type="text/javascript" src="js/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="js/bootstrap.min.js"></script>
<script type="text/javascript" src="js/jquery-scrolltofixed.js"></script>
<script type="text/javascript" src="js/jquery.nav.js"></script> 
<script type="text/javascript" src="js/jquery.easing.1.3.js"></script>
<script type="text/javascript" src="js/jquery.isotope.js"></script>
<script src="js/fancybox/jquery.fancybox.pack.js" type="text/javascript"></script> 
<script type="text/javascript" src="js/wow.js"></script> 
<script type="text/javascript" src="js/custom.js"></script>
<script src="js/publication_script.js"></script>

</body>
</html>